plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/AbstractPinotIntegrationSmokeTest.java

```diff
@@ -294,19 +299,34 @@ protected QueryRunner createQueryRunner()
         pinot.createSchema(getClass().getClassLoader().getResourceAsStream("quotes_in_column_name_schema.json"), QUOTES_IN_COLUMN_NAME_TABLE);
         pinot.addRealTimeTable(getClass().getClassLoader().getResourceAsStream("quotes_in_column_name_realtimeSpec.json"), QUOTES_IN_COLUMN_NAME_TABLE);
 
-        Map<String, String> pinotProperties = ImmutableMap.<String, String>builder()
-                .put("pinot.controller-urls", pinot.getControllerConnectString())
-                .put("pinot.max-rows-per-split-for-segment-queries", String.valueOf(MAX_ROWS_PER_SPLIT_FOR_SEGMENT_QUERIES))
-                .put("pinot.max-rows-for-broker-queries", String.valueOf(MAX_ROWS_PER_SPLIT_FOR_BROKER_QUERIES))
-                .build();
-
         return PinotQueryRunner.createPinotQueryRunner(
                 ImmutableMap.of(),
-                pinotProperties,
+                pinotProperties(pinot),
                 Optional.of(binder -> newOptionalBinder(binder, PinotHostMapper.class).setBinding()
                         .toInstance(new TestingPinotHostMapper(pinot.getBrokerHostAndPort(), pinot.getServerHostAndPort()))));
     }
 
+    protected Map<String, String> pinotProperties(TestingPinotCluster pinot)
+    {
+        ImmutableMap.Builder pinotPropertiesBuilder = ImmutableMap.<String, String>builder()
+                .put("pinot.controller-urls", pinot.getControllerConnectString())
+                .put("pinot.max-rows-per-split-for-segment-queries", String.valueOf(MAX_ROWS_PER_SPLIT_FOR_SEGMENT_QUERIES))
+                .put("pinot.max-rows-for-broker-queries", String.valueOf(MAX_ROWS_PER_SPLIT_FOR_BROKER_QUERIES));
+        if (isControllerSecured()) {
+            pinotPropertiesBuilder
+                    .put("pinot.authentication.controller.type", BASIC_INLINE.name())
+                    .put("pinot.authentication.controller.basic-inline.user", "admin")
+                    .put("pinot.authentication.controller.basic-inline.password", "verysecret");
+        }
+        if (isBrokerSecured()) {
+            pinotPropertiesBuilder
+                    .put("pinot.authentication.broker.type", BASIC_INLINE.name())
+                    .put("pinot.authentication.broker.basic-inline.user", "query")
+                    .put("pinot.authentication.broker.basic-inline.password", "secret");
+        }
+        return pinotPropertiesBuilder.build();
```

Comment:
Please provide secured properties from `TestSecuredPinotIntegrationSmokeTest` (not this class).

Reply:
sure. it's probably easier to discuss over slack... I'll send you a message later today in case there are more changes you'd like me to make